nucypher/blockchain/eth/clients.py

```diff
@@ -429,23 +432,26 @@ def new_account(self, password: str) -> str:
                                                                         password=password)
         return insecure_account
 
-    def sign_transaction(self, transaction: dict) -> bytes:
-        # Get signing key of test account
-        address = to_canonical_address(transaction['from'])
-        signing_key = self.w3.provider.ethereum_tester.backend._key_lookup[address]._raw_key
+    def __get_signing_key(self, account: bytes):
+        """Get signing key of test account"""
```

Comment:
Probably not in the scope of this PR, but we should extract EthereumTester client to the `tests` package.

Reply:
Opened #1807